### ISSUE TITLE
Sometimes "Secure connection" Button Doesn't Do Anything

### ISSUE THREAD

**Issue by djohannes** [mullvad/mullvadvpn-app#2892]
## Issue report

Operating system: iOS 15 Beta

App version: 2021.3-beta2

### Issue description

Since iOS 15 beta (and the TestFlight version of the Mullvad app), I randomly just can't click on "Secure Connection", nothing happens. I haven't found a way to reproduce this yet and when it happens, all I have to do is close the app completely and open it again, suddenly it works. I don't get an error, the button just doesn't do anything and even when I select a new location nothing happens.

**Comment by pronebird** [comment]
Please send the problem report via in-app Settings > Problem report so we could look into logs. Thanks.

**Comment by djohannes** [comment]
@pronebird Done! (I just wrote "#2892" in the description) If the log doesn't include the bug (don't know if I have to send it right after I experienced the bug or if it's still fine a bit later), I can send one more after I've experienced it again.

**Comment by pronebird** [comment]
@djohannes great! we capture logs for the current and previous sessions so ideally to get a useful log, you should submit the problem report either during the current or next run of the app.

**Comment by djohannes** [comment]
@pronebird So I just experienced the same issue again and sent 2 more log reports, one with the current session when I just experienced the bug again and a 2nd one after I restarted the app and suddenly could connect again. 
I've also noticed that most of the time (but not always, it's still kind of random) this happens after I switched to the official WireGuard app and connected and disconnected a different VPN. Again, not always so I'm not sure if it's actually related.
